### ISSUE TITLE
There is no "requirements.txt" in drive folder

### ISSUE THREAD

**Issue by qujixiang** [ucsb-seclab/regulator-dynamic#1]
There is no "requirements.txt" in drive folder.

**Comment by rasoul-j** [comment]
@Qujixiang Were you able to build the tool without the "requirements.txt"?

**Comment by qujixiang** [comment]
> @Qujixiang Were you able to build the tool without the "requirements.txt"?

Yeah, you need check the dependencies, and then install them manually.

**Comment by jeremy-rifkin** [comment]
In case it's of help to anyone else, I put this together to make setup, build, and use easier https://github.com/jeremy-rifkin/regex-fuzzing
